.github/workflows/galata.yml

```
@@ -88,6 +88,19 @@ jobs:
           path: |
             galata/playwright-report
 
+      - name: Update snapshots
+        if: failure()
+        run: |
+          cd galata
+          jlpm run test:update
+
+      - name: Upload updated snapshots
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: galata-updated-snapshots
+          path: galata/test/jupyterlab
```

Select snapshots from all projects and only snapshots & apply prefix `jupyterlab-`

```suggestion
          name: jupyterlab-galata-updated-snapshots
          path: galata/test/**/*-snapshots/*.*
```